src/background.js

```diff
@@ -27,3 +27,31 @@ browser.runtime.onMessage.addListener(async message => {
 
 // GitHub Enterprise support
 injectContentScripts();
+
+browser.contextMenus.create({
+	id: 'enable-extension-on-new-domain',
+	title: 'Enable Refined GitHub on this domain',
+	contexts: ['page_action'],
+	documentUrlPatterns: [
+		'http://*/*',
+		'https://*/*'
+	]
+});
+
+browser.contextMenus.onClicked.addListener(async ({menuItemId}, {tabId, url}) => {
+	/* eslint-disable no-alert */
+	/* global chrome */
+	if (menuItemId === 'enable-extension-on-new-domain') {
+		chrome.permissions.request({
```

Comment:
The browser polyfill doesn't have this one?

Reply:
Sadly, no. Forgot to include the link https://github.com/mozilla/webextension-polyfill/issues/38

Reply:
Can you include the link as a comment in the source? For posterity. Maybe as a `TODO` comment for the future.

Reply:
b85ff71